.npmignore

```diff
@@ -0,0 +1,44 @@
+yarn.lock
```

Comment:
I first wondered why there's so many things ignored that are never part of this repo. Then I saw that it's a copy of the `.gitignore` file. I then understood how it helps with the browser bundles.

Could you please for npm noobs like me add a comment at the top along the lines of "This is a copy of `.gitignore` without ignoring `dist`, please keep it in sync".